app/src/main/AndroidManifest.xml

```diff
@@ -49,6 +49,12 @@
             android:enabled="true" />
 
         <receiver android:name=".Receivers.NotificationAlarmReceiver" />
+        <receiver android:name=".Receivers.NetworkDialogHandler">
```

Comment:
we need to do dynamic registration. A lot of people don't understand the difference, please make sure that you do.